features/admin.py

```diff
@@ -61,6 +62,11 @@ class OverrideInline(TranslatableTabularInline):
     extra = 1
 
 
+class PriceListInLine(TranslatableTabularInline):
```

Comment:
why did you name this `PriceList...` if the model is `PriceTag`? 🤔

Reply:
fixed.